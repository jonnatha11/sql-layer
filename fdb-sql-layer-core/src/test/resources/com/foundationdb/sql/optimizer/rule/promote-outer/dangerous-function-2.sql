SELECT * FROM customers LEFT JOIN orders ON customers.cid = orders.cid
 WHERE IFNULL(orders.order_date, CURRENT DATE) > '2011-01-01'
