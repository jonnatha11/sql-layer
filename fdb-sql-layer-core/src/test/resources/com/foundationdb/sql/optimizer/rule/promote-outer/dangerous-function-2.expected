SelectQuery@3a51127a
  ResultSet@57922f46[cid, name, oid, cid, order_date]
    Project@3e5dc994[customers.cid, customers.name, orders.oid, orders.cid, orders.order_date]
      Select@5e29c58e[ifnull(orders.order_date,current_date()) > 2011-01-01]
        JoinNode@8691dee(LEFT[customers.cid == orders.cid])
          TableSource@2d2bf892(customers)
          TableSource@21cc5069(orders)
